protractor-typescript-cucumber/Config/config.ts

```diff
@@ -0,0 +1,41 @@
+import {ProtractorBrowser} from 'protractor';
+/*
```

Comment:
I think this was addressed with commit https://github.com/angular/protractor-cookbook/pull/14/commits/af051af2605541e55d0d714bd18c9add7fcd21f9